doc/python-grader/index.md

```diff
@@ -0,0 +1,56 @@
+# Python Autograder
+
+This file documents the default Python autograder included in the `prairielearn/grader-python` Docker image.  For general information on how to set up an external grader, visit the [external grading](../externalGrading.md) page. 
+
+## Setting up
+
+### `info.json`
+
+The question should be first set up to enable [external grading](../externalGrading.md), with `"gradingMethod": "External"` set in the `info.json` settings.  To use the specific Python autograder detailed in this document, `"image"` should be set to `"prairielearn/grader-python"` and `"entrypoint"` should point to `"/python_autograder/run.sh"` in the `"externalGradingOptions"` dictionary.
+
```

Comment:
Directly show some kind of example here of an `info.json`.

Reply:
Thinking about this more... We probably should be trying to automatically include files in the PL repository to automatically update documentation. I'll make this an issue for me to revisit going forward.